apisix/plugins/http-logger.lua

```diff
@@ -277,6 +277,8 @@ function _M.log(conf, ctx)
         max_retry_count = conf.max_retry_count,
         buffer_duration = conf.buffer_duration,
         inactive_timeout = conf.inactive_timeout,
+        route_id = ctx.route_id,
```

Comment:
for `route_id`, we should ways use `ctx.var.route_id`

it is the recommended way. and please fix similar points.